aliasing/api/context.py

```diff
@@ -18,6 +18,7 @@ def __init__(self, ctx):
         self._author = AliasAuthor(ctx.author)
         self._prefix = ctx.prefix
         self._alias = ctx.invoked_with
+        self._message = ctx.message.id
```

Comment:
Let's name this attr `_message_id` if it's just saving the ID - the name should make it clear what lives there, and naming it just `_message` makes it seem like it saves the entire Message object.

Same for the public `.message` -> `.message_id` attr.